src/main/java/salesforce/ui/pages/task/AbstractTask.java

```diff
@@ -84,6 +84,34 @@
      */
     protected abstract String getSubject(String subject);
 
+    /**
+     * Sets Contact.
+     *
+     * @param contact value.
+     */
+    protected abstract void setContact(String contact);
```

Comment:
**final** String contact

Reply:
check again

Reply:
this is class abstract!!! checkstyle doesn't display error!